cloudify_agent/tests/installer/test_operations.py

```diff
@@ -95,6 +95,40 @@ def test_local_agent_from_package(self, _):
         env.execute('uninstall', task_retries=1)
         self.wait_for_daemon_dead(name=agent_name)
 
+    @only_os('posix')
```

Comment:
It should, we can rerun it on CI without, but for now, here's the logs from my machine with and without the fix https://gist.github.com/tehasdf/4329cec872067c661d6e226a1eca1c2e